### PR TITLE
[Easy] Add validation to index.js

### PR DESCRIPTION
**Pull request by dfangshuo**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # "Describe what's new in this PR"

Add validation of configuration to fail explicitly when environment is not properly specified

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,4 +3,10 @@ REACT_APP_AIRTABLE_API_KEY=
 UTILITY_API_KEY=
 MAIL_SERVER_EMAIL=
 MAIL_SERVER_PASS=
-ADMIN_EMAIL=
\ No newline at end of file
+ADMIN_EMAIL=
+AIRTABLE_BASE_ID=
+AIRTABLE_ENDPOINT_URL=
```
Comment (aivantg) [inline .env.example:8]: we don't actually need this variable, this is only for airlock

**Review by aivantg — APPROVED** [review]
going to approve and merge, but i'm gonna clean up some env variables in another PR
